codestodians/food-basket-front-end
# [feat: #153992929] Style Cart Component

#### What does this PR do?
- Style the Cart Component such that the badge can be easily controlled
- Ensure the Cart Component is responsive

#### Description of Task to be completed?
- Update react-hot-reloader module
- Changed the font-family
- Add fix for cart component that shows number of items if cart is empty

#### How should this be manually tested?
- Pass a numeric value to badgeContent prop
- Notice the immediate change on the webpage

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#153992929](https://www.pivotaltracker.com/story/show/153992929)
